<?xml version="1.0" encoding="UTF-8"?>
<!--
  -  Copyright (c) 2025  Ioannis Panagiotopoulos
  -
  -  This program is free software: you can redistribute it and/or modify
  -  it under the terms of the GNU General Public License as published by
  -  the Free Software Foundation, either version 3 of the License, or
  -  (at your option) any later version.
  -
  -  This program is distributed in the hope that it will be useful,
  -  but WITHOUT ANY WARRANTY; without even the implied warranty of
  -  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  -  GNU General Public License for more details.
  -
  -  You should have received a copy of the GNU General Public License
  -  along with this program.  If not, see <https://www.gnu.org/licenses/>.
  -->

<ui version="4.0">
 <class>Elpida::Application::CustomBenchmarkView</class>
 <widget class="QWidget" name="CustomBenchmarkView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="glMain">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbAvailableBenchmarks">
     <property name="title">
      <string>Available benchmarks</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lblSelectBenchmark">
        <property name="text">
         <string>Select benchmark:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="twBenchmarks">
        <column>
         <property name="text">
          <string notr="true">Name</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gbExecution">
     <property name="title">
      <string>Execution</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pbRun">
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
